lib/jsanalyze.js

```diff
@@ -196,14 +196,16 @@ exports.analyzeJs = function analyzeJs(contents, opts) {
 			}
 
 			// copy over polyfill and its dependencies
-			// WARNING: REMEMBER TO UPDATE THIS IF 'babel-polyfill' DEPENDENCIES CHANGE!
-			[ 'babel-polyfill', 'core-js', 'regenerator-runtime' ].forEach((moduleName) => {
-				const moduleSrcDir = path.dirname(require.resolve(path.join(moduleName, 'package.json'))),
-					moduleDstDir = path.join(modulesDir, moduleName);
+			// WARNING: REMEMBER TO UPDATE THIS IF '@babel/polyfill' DEPENDENCIES CHANGE!
+			// Because core-js is under @babel/polyfill, it will get copied. require.resolve didn't work here for it anyways
+			[ '@babel/polyfill', 'regenerator-runtime' ].forEach((moduleName) => {
```

Comment:
core-js residing under `@babel/polyfill` seems to be causing issues for me in kitchensink as the require path is `core-js/modules/es7.symbol.async-iterator` which isn't the location as it's under `@babel/polyfill`, nuking the package-lock makes puts core-js back in the top level (but causes a whole host of changes).

I think something like the below for resolving the modules might work, but would require us to bump our min Node to 8.9 for the paths option in require.resolve (so we know we're getting the exact dependency `@babel/polyfill` wants), this makes kitchensink happy.

```js
const polyfillPkgJSONKLocation = require.resolve(path.join('@babel/polyfill', 'package.json'));
const polyfillPath = path.dirname(polyfillPkgJSONKLocation);
const polyfillPkgJSON = fs.readJSONSync(polyfillPkgJSONKLocation);
const polyfillDest =  path.join(modulesDir, '@babel/polyfill');
if (!fs.existsSync(polyfillDest)) {
	fs.copySync(polyfillPath, polyfillDest);
}
for (const dependency of Object.keys(polyfillPkgJSON.dependencies)) {
	const dependencyPath = path.dirname(require.resolve(dependency, { paths: [ polyfillPath ] }));
	const moduleDstDir = path.join(modulesDir, dependency);
	if (!fs.existsSync(moduleDstDir)) {
		fs.copySync(dependencyPath, moduleDstDir);
	}
}
```